marketplace/cmd/info.go

```diff
@@ -0,0 +1,78 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/dustin/go-humanize"
+	"github.com/google/go-github/v56/github"
+	"github.com/gookit/color"
+	"github.com/ignite/apps/marketplace/pkg/xgithub"
+	"github.com/ignite/cli/ignite/pkg/cliui"
+	"github.com/spf13/cobra"
+)
+
+// NewInfo creates a new info command that shows the details of an ignite app.
+func NewInfo() *cobra.Command {
+	return &cobra.Command{
+		Use:   "info [app-name]",
+		Short: "Show the details of an ignite app",
+		Args:  cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			appName := args[0]
+			appName = strings.TrimPrefix(appName, "github.com/")
+			appNameParts := strings.Split(appName, "/")
+			if len(appNameParts) != 2 {
```

Comment:
The app name can be only `mars` instead "github.com/test/mars"

Reply:
The repository name alone is not unique in Github since multiple accounts can have a repository with the same name therefore we can't find it.

Reply:
We will only accept apps inside GitHub? so, we will exclude Gitlab and bitbucket repos for instances